opensuse_buildservice/debian/preinst

```diff
@@ -0,0 +1,70 @@
+#!/bin/sh
+# preinst script for notqmail
+#
+# see: dh_installdeb(1)
+
+set -e
+qmaildir=/var/qmail
+logdir=/var/log/svc
+rm=/bin/rm
+
+# summary of how this script can be called:
+#        * <new-preinst> `install'
+#        * <new-preinst> `install' <old-version>
+#        * <new-preinst> `upgrade' <old-version>
+#        * <old-preinst> `abort-upgrade' <new-version>
+# for details, see http://www.debian.org/doc/debian-policy/ or
+# the debian-policy package
+
+ID=`id -u`
+if [ $ID -ne 0 ] ; then
+	echo "You are not root" >&2
+	exit 1
+fi
+
+case "$1" in
+    install|upgrade)
+	${rm} -f /tmp/notqmail.install /tmp/notqmail.upgrade
+	touch /tmp/notqmail.$1
+	if [ " $1" = " upgrade" ] ; then
+		exit 0
+	fi
+	#
+	# Create a users and groups. Do not report any problems if they already
+	# exists.
+	#
+	(
+	echo "Adding notqmail users/groups"
+	/usr/bin/getent group nofiles   > /dev/null || /usr/sbin/groupadd nofiles  || true
+	/usr/bin/getent group qmail     > /dev/null || /usr/sbin/groupadd qmail    || true
+	
+	for i in alias qmaild qmaill qmailp qmailq qmailr qmails
+	do
+		${rm} -f /var/spool/mail/$i
+	done
+	
+	/usr/bin/getent passwd alias    > /dev/null || /usr/sbin/useradd -M -g nofiles  -d ${qmaildir}/alias  -s /sbin/nologin  alias || true
```

Comment:
Seems like Debain policies prefer ```adduser```

Reply:
I'm not sure and knowledgeable about this. I will check this out.

Reply:
done for addgroup and adduser